lodash.src.js

```diff
@@ -3592,6 +3594,15 @@
         }
         return fn.apply(thisBinding, args);
       }
+
+      if (isAryExact) {
+        var plist = [];
+        for (var i = 0; i < ary; i++) { plist.push('p' + i); }
+        var params = plist.join(',');
+        var fn = wrapper;
+        eval('wrapper = function(' + params + ') { return fn.apply(this, arguments); }');
```

Comment:
Because this is opt-in I don't mind the CSP gotcha. It's similar to `_.template` use. However I would move this into the `_.ary` function directly and not try to incorporate it into the wrapper helpers. I'd also switch `eval` to `Function(..)` use.